docs/Zeta_environment_setup.md

```diff
@@ -206,4 +205,5 @@ TBD
 
 ### Problems
 
-- In the REST API of "add compute instance ports", what do "ips_port", "mac_port", "ip_node" and "mac_node" mean, respectively?
+- I checked DISABLED_2_ports_CREATE_test_traffic_PARENT in /test/gtest/aca_test_ovs_l2.cpp. There is a step to construct a neighbor port. Does our test also need to add a neighbor port?

```

Comment:
add a neighbor port is not needed anymore because we are switching to use generic vxlan outport.